packages/plugin-react/src/index.ts

```diff
@@ -93,7 +108,7 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
           if (isReactModule && filter(id)) {
             useFastRefresh = true
             plugins.push([
-              await import('react-refresh/babel'),
+              await interopDefault(import('react-refresh/babel')),
```

Comment:
I don't think this call is necessary. See [here](https://unpkg.com/browse/react-refresh@0.10.0/babel.js). This module uses CommonJS syntax.

Reply:
Hmm okay. Guess I'm wrong. Seems that the CommonJS module is coerced into an ES `Module` instance by dynamic import.

Reply:
Interestingly, I get this error when doing `import('react-refresh/babel')` in Node REPL
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '~/vite/node_modules/react-refresh/babel' imported from ~/vite/packages/plugin-react/repl
Did you mean to import react-refresh/babel.js?
```
..while `import('react-refresh/babel.js')` works fine.